lib/identity/design.rb

```diff
@@ -72,5 +72,9 @@ class Design < Sinatra::Base
         slim :"design/index"
       end
     end
+
+    def client_deny_url
+      "https://id.heroku.com/#deny"
```

Comment:
I needed this fix to test the authorization endpoint in our design since the view calls this method.
https://github.com/heroku/identity/blob/master/lib/identity/design.rb#L51-L54